awx/ui/client/lib/components/layout/side-nav-item.partial.html

```diff
@@ -1,8 +1,10 @@
 <div class="at-Layout-sideNavItem" ng-click="vm.go()" ng-class="{'is-active': vm.isRoute}"
     ng-show="(!systemAdminOnly || layoutVm.isSuperUser) && layoutVm.isLoggedIn &&
         !layoutVm.licenseIsMissing">
-            <i class="fa {{ iconClass }}"></i>
-            <span class="at-Layout-sideNavItemName" ng-show="navVm.isExpanded">
-                {{ layoutVm.getString(name) }}
-            </span>
+        <at-popover state="vm.tooltip" ng-if="!navVm.isExpanded"></at-popover>
```

Comment:
so I think that the at-popover is used on the credential form as well, so the styles you've added are influencing the positioning of the popovers in that form too: 
![image](https://user-images.githubusercontent.com/7010629/30350260-d6941ebc-97ca-11e7-8c3d-b22e5fdbbffa.png)

Perhaps you could add an additional set of classes for nav-related popovers so that they don't effect the credential popovers?

Reply:
On my last PR of #95 I made the footer relative and removed it's high z-index since now that the footer is not static it doesn't have any sense so after mergin #95 the tooltip should be visible above the footer.

The loading modal on the bottom right corner is visible too on #95